src/plugins/viewLargeAction/viewLargeAction.js

```diff
@@ -0,0 +1,124 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2021, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import PreviewHeader from '@/ui/preview/preview-header.vue';
+
+import Vue from 'vue';
+
+export default class ViewLargeAction {
+    constructor(openmct) {
+        this.openmct = openmct;
+
+        this.cssClass = 'icon-items-expand';
+        this.description = 'View Large';
+        this.group = 'windowing';
+        this.key = 'large.view';
+        this.name = 'Large View';
+        this.priority = 1;
+        this.showInStatusBar = true;
+
+        if (ViewLargeAction.isVisible === undefined) {
+            ViewLargeAction.isVisible = false;
+        }
+    }
+
+    invoke(objectPath, view = {}) {
```

Comment:
must be invoked with a view, right? If so, let's not default it to an empty object.

Reply:
it does not work for  `/src/ui/mixins/context-menu-gesture.js`
https://github.com/nasa/openmct/pull/3770/files#diff-67c705006942b6dfa025e97264c78f9a24b8d845d5b31b6d877a8dca91d65540R39